catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -306,9 +306,8 @@
     <bean id="deprecatedProductCache" class="ddf.catalog.cache.impl.ResourceCacheImpl"
           init-method="setupCache"
           destroy-method="teardownCache">
-        <property name="productCacheDirectory" value=""/>
-        <property name="cacheDirMaxSizeMegabytes" value="10240"/>
```

Comment:
Looks like it might not matter currently if that property is available since it never actually gets used by the `ResourceCacheImpl` at the moment. I'm thinking that refactoring the `ResourceCacheImpl` much more is beyond the scope of this pr, but I will address it in another one shortly